### PR TITLE
Yet another attempt to fix AppVeyor deployment

### PR DESCRIPTION
**Pull request by Mailaender**
Now trying the reverse of https://github.com/OpenRA/OpenRA/pull/7758.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -34,12 +34,13 @@ before_deploy:
   - move /Y %APPVEYOR_BUILD_FOLDER%\packaging\windows\OpenRA.Setup.exe %APPVEYOR_BUILD_FOLDER%\OpenRA-%APPVEYOR_REPO_TAG_NAME%.exe
 
 artifacts:
-  - path: OpenRA-%APPVEYOR_REPO_TAG_NAME%.exe
+  - name: Installer
+    path: OpenRA-%APPVEYOR_REPO_TAG_NAME%.exe
```
Comment (obrakmann) [inline appveyor.yml]: just to be safe, I'd turn those two lines around. That's the way it's always shown in the appveyor docs. Not sure if it makes a difference, but...

Comment (Mailaender) [inline appveyor.yml]: That may be it. Updated.

Comment (obrakmann) [inline appveyor.yml]: And another thing I just saw: it seems that the syntax for env variables in that place is $(envVarName), not %envVarName%. (see the third paragraph on http://www.appveyor.com/docs/packaging-artifacts)

File: appveyor.yml [hunk 2]
```diff
@@ -35,11 +35,12 @@ before_deploy:
 
 artifacts:
   - path: OpenRA-%APPVEYOR_REPO_TAG_NAME%.exe
```
Comment (obrakmann) [inline appveyor.yml]: And another thing I just saw: it seems that the syntax for env variables in that place is $(envVarName), not %envVarName%. (see the third paragraph on http://www.appveyor.com/docs/packaging-artifacts)

Comment (Mailaender) [inline appveyor.yml]: You mean http://www.appveyor.com/docs/packaging-artifacts#pushing-artifacts-from-scripts? We are not pushing from a PowerShell script.

Comment (obrakmann) [inline appveyor.yml]: No, I mean this:
```
You can use wildcards and environment variables in the artifact path. Let's say the "configuration" variable contains the current build configuration. Then to upload all assemblies in bin directory:

myproject\bin\$(configuration)\*.dll
```

Comment (Mailaender) [inline appveyor.yml]: Oh right, see also line 223 of http://www.appveyor.com/docs/appveyor-yml.

**Comment by Mailaender** [comment]
Updated.

**Comment by obrakmann** [comment]
:+1: 
We'll reach a point where we'll never have to touch this again. Have faith in this :p

**Pull request merged**